ivy_tests/test_ivy/test_frontends/test_numpy/test_ndarray/test_ndarray.py

```diff
@@ -4,10 +4,48 @@
 
 # local
 import ivy_tests.test_ivy.helpers as helpers
-from ivy_tests.test_ivy.helpers import handle_cmd_line_args
+from ivy_tests.test_ivy.helpers import handle_cmd_line_args, num_positional_args
 import ivy_tests.test_ivy.test_frontends.test_numpy.helpers as np_frontend_helpers
 
 
+# argmax
+handle_cmd_line_args
```

Comment:
Hey! Good work on the test, this should be `@handle_cmd_line_args`

Reply:
Hi, thanks for spotting that! Fixed it :)

Reply:
Hi, I don't think you committed the changes though, it's still the same. Kindly confirm. Thanks!

Reply:
Hi, it should be fixed now. Let me know if there's anything else I need to change!